### ISSUE TITLE
Dismissing the "Support Sodium!" button breaks the Shader Packs button

### ISSUE THREAD

**Issue by kadenkat** [FlashyReese/reeses-sodium-options#129]
If you click the [X] next to "Support Sodium!" it breaks the "Shader Packs..." button.

It is reproducible by opening the following file:
<path>\ModrinthApp\profiles\Fabulously Optimized\config\sodium-options.json

and setting the notification flag to false:
"has_cleared_donation_button": false,

[issue.webm](https://github.com/user-attachments/assets/87055da6-4cc3-443c-9712-6ab64e4cb2e1)

**Comment by FlashyReese** [comment]
This has been fixed in the latest release.

**Comment by Falkraath** [comment]
No, it has not been fixed - Still reproduceable in 1.8.3 - Minecraft Version 1.21.1 ; Just fixed it by setting the notification flag to false.

Edit: Despite changing the settings being a temporary workaround , i found the true culprit being the ingame Accessibility settings. Disable Background Blur ;) [](https://www.reddit.com/r/feedthebeast/comments/1jm15sy/iris_shader_not_showing/)
